tasks/mpi/mezhuev_m_lattice_torus/perf_tests/main.cpp

```diff
@@ -0,0 +1,94 @@
+#include <gtest/gtest.h>
+
+#include <boost/mpi/timer.hpp>
+#include <cmath>
+#include <vector>
+
+#include "core/perf/include/perf.hpp"
+#include "mpi/mezhuev_m_lattice_torus/include/mpi.hpp"
+
+TEST(mezhuev_m_lattice_torus, perf_test_pipeline_run) {
+  boost::mpi::communicator world;
+
+  int grid_size = std::sqrt(world.size());
+  if (grid_size * grid_size != world.size() || world.size() < 4) {
+    return;
+  }
+
+  std::vector<uint8_t> input_data(2020, 1);
```

Comment:
Вместо магического числа в тестах использовать константу
```suggestion
  const int count = 2020;
  std::vector<uint8_t> input_data(count, 1);
```
Сделать во всех тестах